components/src/components/FormGenerator/FormGenerator.tsx

```diff
@@ -227,7 +228,7 @@ export const FormGenerator = (props: FormGeneratorProps) => {
         if (isFormGeneratorRow(obj)) {
           if (obj.rowType === 'button') {
             return (
-              <GridChild columnsOccupied="all">
+              <GridChild columnsOccupied="all" style={buttonRowStyle}>
```

Comment:
Jeg hadde ikke brukt inline style her. Jeg foreslår å lage en styled component i `FormGenerator.styles.tsx` som styler videre `<GridChild>`, f.eks: 
```
export const ButtonRow = styled(GridChild)`
  margin-top: ${tokens.buttonRow.marginTop};
`;
```

og da kan du bruke den slik: 

```suggestion
              <ButtonRow columnsOccupied="all">
```
Det blir også med oversiktlig med godt definert knapperad.

Reply:
Fixed.